app/helpers/checklist_tests_helper.rb

```diff
@@ -59,6 +59,19 @@ def coded_attribute?(criteria)
   def lookup_valueset_name(oid)
     vs = HealthDataStandards::SVS::ValueSet.where(oid: oid)
     return oid unless vs && vs.first
-    "#{vs.first.display_name}: #{oid}"
+    "#{vs.first.display_name}"
+  end
+
+
+  def lookup_valueset_oid(oid)
+    vs = HealthDataStandards::SVS::ValueSet.where(oid: oid)
+    return oid unless vs && vs.first
+    "#{oid}"
```

Comment:
That is what I want it to do. Due to the new CSS changes that Eny wanted me to add, the oid and the valueset need to be separated out into different li tags. The other function returned both the oid and the valueset name together so i created a helper function that just displayed the oid.